site/app/controllers/student/SubmissionController.php

```diff
@@ -1071,7 +1071,6 @@ public function ajaxGetGradeableValues(string $gradeable_id): JsonResponse {
         $graded_gradeable = $this->core->getQueries()->getGradedGradeable(
             $gradeable,
             $user_id,
-            $gradeable->isTeamAssignment()
```

Comment:
The third argument of the query is the team name, so this boolean will never have an effect. It likely should be updated to have a correct value, but this change will not cause a regresssion